includes/defaults.inc.php

```diff
@@ -458,6 +458,7 @@ function set_debug($debug) {
 // Ignore lines starting with #
 // $config['collectd_dir']                 = '/var/lib/collectd/rrd';
 // $config['smokeping']['dir']             = "/var/lib/smokeping/";
+$config['smokeping']['pings']             = 10;
```

Comment:
Why the change in default behavior and not keeping the `old` value like people are used to?